test/functional/ui/spell_spec.lua

```diff
@@ -0,0 +1,41 @@
+-- Test for scenarios involving 'spell'
+
+local helpers = require('test.functional.helpers')(after_each)
+local Screen = require('test.functional.ui.screen')
+local feed, insert, eq, eval, clear, expect = helpers.feed, helpers.insert, 
+  helpers.eq, helpers.eval, helpers.clear, helpers.feed_command, helpers.expect
+local Screen = require('test.functional.ui.screen')
+local feed_command = helpers.feed_command
+
+describe('combining long lines', function()
+  local screen
+
+  before_each(function()
+      clear()
```

Comment:
Indentation, also with `screen:snapshot_util()` there should be no reason to not test highlighting.

Reply:
Not sure how to test highlighting.  I'll try to get it added.

Reply:
If you use `snapshot_util()` it should give the highlights, then they can be moved to a `screen:set_default_attr_ids({...}}` call. See e g `test/functional/ui/bufhl_spec.lua`